pkg/snapshot/restorer/restorer_test.go

```diff
@@ -436,6 +468,82 @@ var _ = Describe("Running Restorer", func() {
 				// behavior of etcd restart.
 			})
 		})
+
+		Context("when snapshots are compressed using different compressionPolicy", func() {
```

Comment:
Can you please add one more case where the base full snapshot itself is compressed?

Reply:
added  the test case.... please have a look